core/include/gnuradio-4.0/Scheduler.hpp

```diff
@@ -85,7 +85,7 @@ class SchedulerBase : public lifecycle::StateMachine<Derived> {
 
     void
     stateChanged(lifecycle::State newState) {
-        emitMessage(msgOut, { { kind::SchedulerStateUpdate, std::string(magic_enum::enum_name(newState)) } });
+        emitMessage(msgOut, { { key::Kind, kind::SchedulerStateUpdate }, { key::What, std::string(magic_enum::enum_name(newState)) } });
```

Comment:
Thinking about it, maybe `key::Data` instead of `key::What` should be used? @ivan-cukic do you have an opinion?

Reply:
done